ruma-client-api/src/r0/session/login.rs

```diff
@@ -40,7 +40,8 @@ ruma_api! {
         pub user_id: UserId,
 
         /// An access token for the account.
-        pub access_token: String,
+        #[serde(skip_serializing_if = "Option::is_none")]
```

Comment:
I don't think this is optional, how would a client work without this or to put it differently, what is the point of logging in if a successful response doesn't give you a access token and thus session on the server?